src/CSSList/CSSList.php

```diff
@@ -296,6 +296,21 @@ public function splice($iOffset, $iLength = null, $mReplacement = null)
         array_splice($this->aContents, $iOffset, $iLength, $mReplacement);
     }
 
+    /**
+     * Inserts an item before its sibling.
```

Comment:
We should also document what happens if the sibling is not in this list (i.e., the item gets appended).

Reply:
Done.